docs/pages/foundations/iconography/ICON_DATA.json

```diff
@@ -2,11 +2,13 @@
   "icons": [
     {
       "name": "add",
-      "categories": "Add"
+      "categories": "Add",
+      "description": "Use this icon to indicate the ability to create or add content"
```

Comment:
Maybe it's more apparent when seeing all the messages together, but "use this icon" seems a bit repetitive and wordy. Perhaps just "Indicates the ability to…"?

Reply:
Hmm I'll check with Cintia on this one and let Layla know